src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -199,6 +199,28 @@ def _encode_subject_alt_name(backend, san):
             )
             gn.type = backend._lib.GEN_IPADD
             gn.d.iPAddress = ipaddr
+        elif isinstance(alt_name, x509.OtherName):
+            gn = backend._lib.GENERAL_NAME_new()
+            assert gn != backend._ffi.NULL
+            other_name = backend._lib.OTHERNAME_new()
+            assert other_name != backend._ffi.NULL
+
+            type_id = backend._lib.OBJ_txt2obj(
+                alt_name.type_id.dotted_string.encode('ascii'), 1
+            )
+            assert type_id != backend._ffi.NULL
+            data = backend._ffi.new("unsigned char[]", alt_name.value)
+            data_ptr_ptr = backend._ffi.new("unsigned char **")
+            data_ptr_ptr[0] = data
```

Comment:
Because despite being a ** we actually do have to provide the data. It's a call to a `d2i` function so you pass the bytes and it returns an object.

The function signature is weird because it should obviously just be a * but that's not how it was defined for some reason.